ansible/roles/couchdb/tasks/main.yml

```diff
@@ -75,6 +75,7 @@
 
 - name: Start CouchDB service
   service: name=couchdb state=started
+  tags: after-reboot
```

Comment:
This will work for couch 1 (swiss) but not couch 2 (icds). confusing we currently have two couch roles for the moment